src/components/App/App.js

```diff
@@ -32,9 +40,31 @@ class App extends React.Component {
 		}
 	};
 
+	showNotification = message => {
+		this.setState(
+			{notificationMessage: message, showNotification: true},
+			() => {
+				setTimeout(() => {
+					this.setState({showNotification: false});
+				}, 5000);
```

Comment:
It'd be good to extract this to a constant, we're using it in several places and would allow us to only have to change one value